tidb.go

```diff
@@ -242,11 +243,11 @@ func RegisterStore(name string, driver kv.Driver) error {
 //    boltdb:///absolute/path
 //
 // The engine should be registered before creating storage.
-func NewStore(path string) (kv.Storage, error) {
-	return newStoreWithRetry(path, util.DefaultMaxRetries)
+func NewStore(path string, security config.Security) (kv.Storage, error) {
```

Comment:
config can be get by `global.Config()` in TiDB, is it necessory to change store interface?